doc/specs/stdlib_bitsets.md

```diff
@@ -0,0 +1,2000 @@
+---
+title: Bitsets
+---
+
+# The `stdlib_bitsets` module
+
+[TOC]
+
+## Introduction
+
+The `stdlib_bitsets` module implements bitset types. A bitset is a
+compact representation of a sequence of `bits` binary values. It can
+equivalently be considered as a sequence of logical values or as a
+subset of the integers 0 ... `bits-1`. The bits are indexed from 0 to
```

Comment:
> or as a subset of the integers 0 ... `bits-1`.

This confuses me. The equivalent integer values are 0 and 1, not 0 through `bits-1`, correct?

Do you mean to say something like this?

"It can equivalently be considered as a sequence of logical values or as a sequence of integers 0 and 1 with indices in the range 0... `bits-1`?"